### PR TITLE
[14.0][mig]-account_analytic_post_by_type

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
LG just please rebase befor ethe merge, the file deleted in analytic_plan is already deleted

**Comment by JuanyDForgeflow** [comment]
@AaronHForgeFlow rebase done

File: account_analytic_post_by_type/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Copyright 2017 ForgeFlow S.L.
```
Comment (AaronHForgeFlow) [inline account_analytic_post_by_type/tests/__init__.py:1]: Please remove this form init files

File: account_analytic_post_by_type/README.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  loss account, in case that there's an analytic account associated
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Eficent <http://www.eficent.com>
```
Comment (AaronHForgeFlow) [inline account_analytic_post_by_type/README.rst]:
```suggestion
* ForgeFlow <http://www.forgeflow.com>
```

**Review by AaronHForgeFlow — APPROVED** [review]
Please do those minor changes before the merge. Thank you!

**Comment by AaronHForgeFlow** [comment]
again, commit history is fine, it is just github website not showing it correctly in the website. merging

**Pull request merged**